modules/convert/convert.go

```diff
@@ -414,3 +415,32 @@ func ToLFSLock(l *git_model.LFSLock) *api.LFSLock {
 		},
 	}
 }
+
+// ToChangedFile convert a gitdiff.DiffFile to api.ChangedFile
+func ToChangedFile(f *gitdiff.DiffFile, repo *repo_model.Repository, commit string) *api.ChangedFile {
+	status := "changed"
+	if f.IsDeleted {
+		status = "deleted"
+	} else if f.IsCreated {
+		status = "added"
+	} else if f.IsRenamed && f.Type == gitdiff.DiffFileCopy {
+		status = "copied"
+	} else if f.IsRenamed && f.Type == gitdiff.DiffFileRename {
+		status = "renamed"
+	}
```

Comment:
if additions and deletions are 0 it should be `unchanged` (for example if file attributes are only changed)